status-core/build.gradle

```diff
@@ -2,12 +2,21 @@ indeedOss.activateFeature 'library'
 indeedLibrary.name = 'status-core'
 
 dependencies {
-    implementation libs.log4j
+    implementation libs.slf4jApi
```

Comment:
nit: I know that we use only implementation internally, but we are explicitly exposing these in the API. Shouldn't they be `api` dependencies?

Reply:
I don't believe this is actually exposed by the api. the log4j1 Logger is exposed in the api, but that one is left out on purpose. The slf4j Logger isn't exposed anywhere.